.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,44 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}

```

Comment:
does this run in all branches? so dev branches too? isn't deploying every commit a bit overkill?

Reply:
Je ne vois pas non plus le problème avec cela comme celui va nous aider à détecter les erreurs ... 🧐

Reply:
if our resources are not limited sure thing👍🏻

Reply:
The strength of the containers is precisely that they are light!

Why didn’t you inquire about it when you were using it?